### PR TITLE
[CDE-562][CLEANUP]

### PR DESCRIPTION
**Pull request by afrjorge**

### CODE REVIEW

**Comment by afrjorge** [comment]
@pamval please review, should be merged along with webdetails/cde#434

File: cdf-core/cdf/js-lib/require-dashboard/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/*!
```
Comment (pamval) [inline cdf-core/cdf/js-lib/require-dashboard/dashboard.js]: This doesn't seem documented according to jsdocs rules. Please review

**Comment by afrjorge** [comment]
@pamval please review, added documentation for the dashboard loader plugin

**Comment by afrjorge** [comment]
@pamval please review, renamed the "dashboard" loader plugin to "dash" to avoid confusion with the Dashboard class, and updated more documentation, should me merged along with webdetails/cde#435

**Pull request merged**